code/datums/elements/strippable.dm

```diff
@@ -284,7 +290,12 @@
 
 /// A utility function for `/datum/strippable_item`s to start unequipping an item from a mob.
 /proc/start_unequip_mob(obj/item/item, mob/source, mob/user, strip_delay)
-	if (!do_mob(user, source, strip_delay || item.strip_delay, ignorehelditem = TRUE))
+	var/strip_mod
+	var/obj/item/clothing/gloves/gloves = user.get_item_by_slot(ITEM_SLOT_GLOVES)
+	if(istype(gloves))
+		strip_mod = gloves.strip_mod
+
+	if (!do_mob(user, source, strip_mod || strip_delay || item.strip_delay, ignorehelditem = TRUE))
```

Comment:
should probably be like this since strip_mod is a divisor to strip time
```suggestion
	var/strip_mod = 1
	var/obj/item/clothing/gloves/gloves = user.get_item_by_slot(ITEM_SLOT_GLOVES)
	if(istype(gloves))
		strip_mod = gloves.strip_mod

	if (!do_mob(user, source, (strip_delay || item.strip_delay) / strip_mod, ignorehelditem = TRUE))
```

Reply:
*I just gotta say, this will definitely runtime if one of them is either* `null` *or* `0`